2021-04-15 19:14:07,826 INFO    MainThread:2593 [wandb_setup.py:_flush():69] setting env: {}
2021-04-15 19:14:07,827 INFO    MainThread:2593 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-15 19:14:07,828 INFO    MainThread:2593 [wandb_init.py:_log_setup():336] Logging user logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210415_191407-119l0aq8/logs/debug.log
2021-04-15 19:14:07,828 INFO    MainThread:2593 [wandb_init.py:_log_setup():337] Logging internal logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210415_191407-119l0aq8/logs/debug-internal.log
2021-04-15 19:14:07,828 INFO    MainThread:2593 [wandb_init.py:init():369] calling init triggers
2021-04-15 19:14:07,828 INFO    MainThread:2593 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-15 19:14:07,829 INFO    MainThread:2593 [wandb_init.py:init():418] starting backend
2021-04-15 19:14:07,829 INFO    MainThread:2593 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-15 19:14:07,875 INFO    MainThread:2593 [backend.py:ensure_launched():123] starting backend process...
2021-04-15 19:14:07,910 INFO    MainThread:2593 [backend.py:ensure_launched():127] started backend process with pid: 2662
2021-04-15 19:14:07,913 INFO    MainThread:2593 [wandb_init.py:init():423] backend started and connected
2021-04-15 19:14:07,917 INFO    MainThread:2593 [wandb_init.py:init():465] updated telemetry
2021-04-15 19:14:07,918 INFO    MainThread:2593 [wandb_init.py:init():484] communicating current version
2021-04-15 19:14:09,917 INFO    MainThread:2593 [wandb_init.py:init():489] got version response 
2021-04-15 19:14:09,917 INFO    MainThread:2593 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-15 19:14:10,065 INFO    MainThread:2593 [wandb_init.py:init():522] starting run threads in backend
2021-04-15 19:14:13,423 INFO    MainThread:2593 [wandb_run.py:_console_start():1524] atexit reg
2021-04-15 19:14:13,428 INFO    MainThread:2593 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-15 19:14:13,429 INFO    MainThread:2593 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-15 19:14:13,432 INFO    MainThread:2593 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-15 19:14:13,432 INFO    MainThread:2593 [wandb_init.py:init():546] run started, returning control to user process
2021-04-15 19:15:43,137 INFO    MainThread:2593 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-15 19:15:43,143 INFO    MainThread:2593 [wandb_run.py:_restore():1466] restore
2021-04-15 19:15:45,302 INFO    MainThread:2593 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 555
  total_bytes: 555
}

2021-04-15 19:15:46,258 INFO    MainThread:2593 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 555
  total_bytes: 5879
}

2021-04-15 19:15:46,419 INFO    MainThread:2593 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 555
  total_bytes: 5879
}

2021-04-15 19:15:46,521 INFO    MainThread:2593 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5879
  total_bytes: 5879
}

2021-04-15 19:15:46,624 INFO    MainThread:2593 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5879
  total_bytes: 5879
}

2021-04-15 19:15:46,727 INFO    MainThread:2593 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5879
  total_bytes: 5879
}

2021-04-15 19:15:46,829 INFO    MainThread:2593 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5879
  total_bytes: 5879
}

2021-04-15 19:15:46,932 INFO    MainThread:2593 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5879
  total_bytes: 5879
}

2021-04-15 19:15:48,366 INFO    MainThread:2593 [wandb_run.py:_show_files():1838] logging synced files
